### PR TITLE
BE: Add Java sections

### PR DESCRIPTION
**Pull request by Noeell**
### What?
Implemented the sections that are now possible to implement
### Anything Else?
closes: #140
closes: #151 
closes: #222 
closes: #469 
closes: #522 
closes: #534 
closes: #535
closes: #537  
closes: #538 
closes: #169
closes: #151 
closes: #471 
closes: #536

### CODE REVIEW

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section511.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                        Your arcane knowledge tells you much about the undead.
+                        Ghouls are known to eat the flesh of the dead as well as the
+                        living. They like to make their homes in crypts and graveyards
+                        and they never venture out during the day, as sunlight burns
+                        their pallid undead flesh. Also, they cannot abide a powder of
+                        salt and iron filings mixed together.
+                        """)
+// TODO: You can purchase these ingredients for 15 Shards at many a
+//market stall. If you make the purchase, note you have salt and
+//iron filings on your Adventure Sheet.
```
Comment (keykey7) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section511.java:30]: I read this as just text... this is just informational with the meaning: if you want to set out for a crypt in the future, remember to buy salt... has no other effect on the game however...

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java [hunk 2]
```diff
@@ -140,6 +140,14 @@ public boolean hasCodeword(String codeword) {
         return codeWords.contains(codeword);
     }
 
+    public boolean hasProfession(String profession) {
+        return profession().toString().equalsIgnoreCase(profession);
```
Comment (keykey7) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java:144]: to string? why not keep it an enum?

**Review by keykey7 — APPROVED** [review]

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java (hunk 2 above)
Comment (Nilstrieb) [reply fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java:144]: yes, please use the `ProfessionEnum` here

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section511.java (hunk 1 above)
Comment (Noeell) [reply fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section511.java:30]: ok, thanks!

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java (hunk 2 above)
Comment (Noeell) [reply fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/PlayerDto.java:144]: good input, I deleted the method and used the enum for checks

**Review by Lion-alt — APPROVED** [review]
LGTM 👍

**Review by oliver-jenny — APPROVED** [review]
Lgtm - look at @keykey7‘s commit regarding staminaWhenUnwounded

**Comment by Lion-alt** [comment]
Resolved change with initialStamina in addStamina and made place for removeStamina, as with a min value on addStamina both methods are needed.

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section520.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                                .choice(d -> d.text("Follow the river north"),
+                                        d -> d.clickableTurnTo(576))
+                                .choice(d -> d.text("Follow the river south"),
+                                        d -> d.clickableTurnTo(82))
+                                .choice(d -> d.text("Head east into the countryside"),
+                                        d -> d.clickableTurnTo(278))
+                                .choice(d -> d.text("West to the main road"),
+                                        d -> d.clickableTurnTo(558))
+                        ));
+        //TODO: YOU DIED
```
Comment (Desi7) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section520.java:40]: If you like you could show the text "you died" and navigate the user back to section 1
Else its looking good to me 👍

**Review by Desi7 — APPROVED** [review]

**Review by Nilstrieb — APPROVED** [review]
amazing!

**Review by JJoeeyy — APPROVED** [review]
LGTM

**Pull request merged**